src/main/java/org/assertj/core/internal/Numbers.java

```diff
@@ -261,6 +259,15 @@ public void assertIsNotCloseToPercentage(final AssertionInfo info, final NUMBER
 
   protected abstract NUMBER absDiff(final NUMBER actual, final NUMBER other);
 
+  protected boolean isNanOrInfinity(final NUMBER value) {
+    if (value instanceof Float)
+      return ((Float) value).isInfinite() || ((Float) value).isNaN();
+    else if (value instanceof Double)
+      return ((Double) value).isInfinite() || ((Double) value).isNaN();
+    else
+      return false;
+  }
```

Comment:
this should go in `RealNumbers` and then you can use it `ComparatroFactory` (please avoid code duplication)

Reply:
why is this comment not addressed @YeeTone ?